scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -255,8 +267,19 @@ Param (
     [ValidateSet("true", "false")]
     [string] $MqttSettingsEnabled = "true",
 
-    [switch] $BypassEdgeInstallation
+    [ValidateNotNullOrEmpty()]
+    [string] $AnalyzerLaEnabled = $(Throw "Analyzer Log Analytic enable string is required"),
```

Comment:
Please refer to what "$MqttSettingsEnabled" does.

Reply:
Per discussion in person, these params will be made into optional parameters.